src/UI/Art/SpriteArtist/SpriteOptions.java

```diff
@@ -17,12 +18,13 @@
   private static final int DRAWSCALE_DEFAULT = 2;
   private static int drawScale = DRAWSCALE_DEFAULT;
 
-  private static Dimension dimensions = new Dimension(WINDOWWIDTH_DEFAULT*drawScale, WINDOWHEIGHT_DEFAULT*drawScale);
+  private static Dimension dimensions = new Dimension(WINDOWWIDTH_DEFAULT * drawScale, WINDOWHEIGHT_DEFAULT * drawScale);
 
   // Set up configurable options.
   private static GraphicsOption drawScaleOption = new GraphicsOption("Draw Scale", 1, 6, DRAWSCALE_DEFAULT);
   private static GraphicsOption dummyOption = new GraphicsOption("Dummy option");
-  private static GraphicsOption[] allOptions = {drawScaleOption, dummyOption};
+  private static GraphicsOption damageStrategyOption = new GraphicsOption("Damage Strategy", Weapon.stratDescriptions, 0);
```

Comment:
Instead of "Strategy", maybe "System", or "Engine". I know you are using the Strategy Pattern, but that won't be clear to people who aren't into OO lingo.